examples/crypto-websockets/server/asyncapi.yaml

```diff
@@ -0,0 +1,38 @@
+asyncapi: 2.4.0
+info:
+  title: asyncapicoin server
+  version: 1.0.0
+  description: |
+    This app creates a dummy server that would stream the price of a fake crypto currency
+servers:
+  websocket:
+    url: ws://localhost:3000
+    protocol: ws
+    x-kind: local
```

Comment:
Let's move this property out of the server definition because that's how it will be done in v3.0.0. For now, we can just use our own extension like this:

```suggestion
x-localServers:
  - websocket
```

This will change the logic in the `registerAdapters.ts` file.